crates/ark/src/reticulate.rs

```diff
@@ -53,7 +59,8 @@ impl ReticulateService {
         let event = CommManagerEvent::Opened(
             service.comm.clone(),
             json!({
-                "input": input
+                "input": input,
+                "reticulate_id": (*RETICULATE_ID).clone(),
             }),
         );
```

Comment:
I'm guessing the comm is always opened from the backend, never the frontend?

Reply:
Yes, that's correct. We only open the comm from the backend when the user calls 'reticulate::repl_python()`.